code/modules/mob/living/simple_animal/simple_animal_vr.dm

```diff
@@ -92,21 +96,39 @@
 /mob/living/simple_animal/PunchTarget()
 	ai_log("vr/PunchTarget() [target_mob]", 3)
 
-	// For things we don't want to eat, call the sideways "parent" to do normal punching
-	if(!vore_active || !will_eat(target_mob))
+	// If we're not hungry, call the sideways "parent" to do normal punching
+	if(!vore_active)
 		return ..()
 
-	// If target is standing we might pounce and eat them
-	if(target_mob.canmove && prob(vore_pounce_chance))
-		target_mob.Weaken(5)
-		target_mob.visible_message("<span class='danger'>\the [src] pounces on \the [target_mob]!</span>!")
-
-	// If they're down or we can eat standing, do it
-	if(!target_mob.canmove || vore_standing_too)
+	// If target is standing we might pounce and knock them down instead of attacking
+	if(target_mob.canmove && prob(vore_pounce_chance) && !issilicon(target_mob) && (world.time > vore_pounce_cooldown)) //pouncing is worth doing, we want to pounce and are not in cooldown. So attempt a pounce!
+		if(will_eat(target_mob) && vore_standing_too) //creatures that can eat you on the spot don't care how healthy you are and get an autohit
```

Comment:
*scream

Make a can_pounce_target() or something return all this garbage. This if-chain is too scary. Maybe have it return false if the chance is real low. (Less than 20?)